portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,6 +15,8 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
Looks like you are not using any array here just yet if not remove the unnecessary import on this pull request to keep the changes self-contained.